bitcoin/BRWalletManagerEvent.c

```diff
@@ -102,9 +103,10 @@ static BREventType bwmSignalTxDeletedEventType = {
 
 extern void
 bwmSignalTxDeleted (BRWalletManager manager,
-                    UInt256 hash) {
+                    UInt256 hash,
+                    int recommendRescan) {
     BRWalletManagerWalletTxDeletedEvent message =
-    { { NULL, &bwmSignalTxDeletedEventType}, manager, hash};
+    { { NULL, &bwmSignalTxDeletedEventType}, manager, hash, recommendRescan};
     eventHandlerSignalEvent (manager->handler, (BREvent*) &message);
 }
```

Comment:
Ignoring `notifyUser`?  That is fine (added later if needed).

Reply:
The `notifyUser` actually isn't tied to the rescan. It seems to be an indicator if if the user should be informed of the deletion (i.e. the logic sometimes has `notifyUser` set but `recommendRescan` not).

In our case, we announce every deletion (and every addition) to the CWM.